platformio_esp32_envs.ini

```diff
@@ -28,7 +28,7 @@ build_flags               = ${mqtt_flags.build_flags}
 [env:custom_ESP32_4M316k]
 extends                   = esp32_common
 platform                  = ${esp32_common.platform}
-build_flags               = ${esp32_common.build_flags}   -DPLUGIN_BUILD_CUSTOM
+build_flags               = ${esp32_common.build_flags}   -DPLUGIN_BUILD_CUSTOM -DESP32 -DHAS_ETHERNET
```

Comment:
You should not need to define `ESP32` here.
Do you have an example where it is needed?

Reply:
Yes, i've accidentialy left it here, need to be removed.
BTW: For what this preprocessor define is used for?
I can't find any place where it is defined, none of envs include this

Reply:
ESP32 is defined in the core libs if I'm not mistaken.